iianmelo/citigames-squadian
# Feat/component/sidebar

What I did
- Sidebar component
- Sidebar buttons and functionality
- Sidebar CITi message

How to test
    1    Open the terminal and go to the folder of this project
    2    In terminal:
    i    git checkout main
    ii    git pull
    iii    git checkout <feat/component/sidebar>
    iv    git pull origin <feat/component/sidebar>
    v    yarn install
    vii    yarn dev
    3    In your browser go to http://localhost:3000/
    4    Check if the component looks just like the mockup in different screen sizes
    5    In VScode open <relative path>
    6    Things to change in the code during the test
if (isAwesome){
    7      return <Component />
    8    }

    9    Read the changed files section on GitHub
    10    Test the things that you think are worth testing, even the ones that are not in this description
